CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 See full changelog for the OpenAPI schema (OAS) [here](https://github.com/plaid/plaid-openapi/blob/master/CHANGELOG.md).
 
 # 14.8.0
```

Comment:
should we be doing a new version bump instead?

Reply:
like a major version?

Reply:
i did a scan of the openapi changes and didn't think so?

Reply:
omg im stupid